frontend/src/components/House/House.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import {css} from 'aphrodite';
+import S from './styles';
```

Comment:
I would prefer `styles` instead of `S`

Reply:
You actually beat me to it but I said exactly the same thing.

Reply:
yes ok np